contexts/DonationContext/src/UseCases/AddDonation/AddDonationUseCase.php

```diff
@@ -60,7 +60,7 @@ public function addDonation( AddDonationRequest $donationRequest ): AddDonationR
 
 		$donation = $this->newDonationFromRequest( $donationRequest );
 
-		if ( $this->policyValidator->needsModeration( $donationRequest ) ) {
+		if ( !$donation->hasExternalPayment() && $this->policyValidator->needsModeration( $donationRequest ) ) {
```

Comment:
Yeah, I was thinking about that, too. But then it felt awkward to have a `needsModeration` that itself calls `needsModeration`. Let me fix that quickly.